docs-website/yarn.lock

```diff
@@ -0,0 +1,10318 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
this file is huge!

Reply:
It's autogenerated and pins all dependencies. It is recommended to be checked in the version control so that the exact same version is used by the CI.